internal/resources/configuration_resource.go

```diff
@@ -271,6 +265,8 @@ func (r *ConfigurationResource) Read(ctx context.Context, req resource.ReadReque
 		schema.Name = types.StringValue(configuration.Spec.ConfigurationFormat.Name)
 		schema.Version = types.StringValue(configuration.Spec.ConfigurationFormat.Version)
 
+		schema.Scope = data.Schema.Scope
```

Comment:
I think we should copy the scope from `configuration.Spec.ConfigurationFormat.Scope` here

Reply:
fixed